### PR TITLE
Add special flow if user is in state w/o protections.

### PR DESCRIPTION
**Pull request by toolness**
If the user is in a state without protections, we need to inform them during the KYR step and let them know that they can create an account and we'll inform them when their state has better protections, but they can't send a letter right now.  We also link out to the FAQ section on "STATES WITH LIMITED PROTECTIONS" so the user can learn more.

## To do

- [x] Add links to FAQ pages for more info.

### CODE REVIEW

File: frontend/lib/norent/letter-builder/know-your-rights.tsx [hunk 1]
```diff
@@ -4,49 +4,93 @@ import Page from "../../ui/page";
...
+  const scf = session.norentScaffolding;
 
-  // Content from national metadata:
-  const legislation = getNorentMetadataForUSState(state)?.lawForBuilder
-    ?.textOfLegislation;
+  if (!scf?.state) {
+    return (
+      <p>
+        Please <Link to={props.prevStep}>go back and choose a state</Link>.
+      </p>
```
Comment (toolness) [inline frontend/lib/norent/letter-builder/know-your-rights.tsx:73]: This is very makeshift and will go away once I implement the whole "you're not supposed to be here!" thing for pages that we don't have enough session info to render meaningful content with.

Comment (sraby) [reply frontend/lib/norent/letter-builder/know-your-rights.tsx:73]: cool cool, good to know

File: frontend/lib/norent/letter-builder/know-your-rights.tsx [hunk 2]
```diff
@@ -4,49 +4,93 @@ import Page from "../../ui/page";
...
+          Learn more.
+        </Link>
+      </p>
+
+      <div className="content">
+        <p>
+          We’ve partnered with <PartnerLink {...props.partner} /> to provide
+          additional support.
+        </p>
+      </div>
```
Comment (sraby) [inline frontend/lib/norent/letter-builder/know-your-rights.tsx:35]: In the figma mock-ups, there was a third paragraph here that said ```If you’d still like to create an account, we can send you updates in the future.``` Was that removed?

**Comment by sraby** [comment]
This looks good Mr. @toolness! Besides my one comment about content stuff, I wonder if there would be an easy way to update the content of the subsequent onboarding steps for Georgia flow users (maybe even just by checking to see if the user's state is a `stateWithoutProtections` in the national metadata) so that instead of referencing the "letter" we could reference their hope of getting future updates. I know it's been a long haul on this so I don't think it's a requirement but lmk— also would be happy to try and add that in myself. Unless there is a particular thing i'm missing that would make that especially hard?

File: frontend/lib/norent/letter-builder/know-your-rights.tsx (hunk 2 above)
Comment (toolness) [reply frontend/lib/norent/letter-builder/know-your-rights.tsx]: oops sorry forgot, just added it!

**Comment by toolness** [comment]
Ahh yeah shoot!  Ok I added alternate copy for all the other onboarding steppes, let me know what you think!

**Pull request merged**